lib/BaseClient.ts

```diff
@@ -35,6 +39,18 @@ abstract class BaseClient extends EventEmitter {
   protected setStatus(status: ClientStatus): void {
     this.logger.info(`${this.constructor.name} status: ${ClientStatus[status]}`);
     this.status = status;
+    this.checkTimers();
+  }
+  private checkTimers() {
+    if (this.status === ClientStatus.ConnectionVerified) {
+      this.updateCapacityTimer = setInterval(async () => {
+        this.maximumOutboundCapacity = (await this.channelBalance()).balance;
```

Comment:
I think we would want some error handling here, right? In case the call fails, I'm thinking we should catch the error, log it,  and mark the client as disconnected. As is I think an exception could crash the process.

If you want to put the part where it marks the client as disconnected off for another PR, that's fine, because when I'm looking at it we'd need to reenable the reconnect timer as well and right now that's done manually only in LndClient. Ideally there should be a base method that we call to mark the client disconnected and enable the reconnection timer, and Lnd & Raiden can have their own logic for how to check for reconnection. It's not trivial and a little outside the scope of this PR.

Reply:
It was indeed crashing the process. Added error handling and logging + TODO for marking the client as disconnected.